test/app-tap/datetime.test.lua

```diff
@@ -513,7 +513,8 @@ test:test("Multiple tests for parser (with nanoseconds)", function(test)
     for _, value in ipairs(tests) do
         local str, epoch, nsec, tzoffset, check
         str, epoch, nsec, tzoffset, check = unpack(value)
-        local dt = date.parse(str)
+        local dt, parsed_syms = date.parse(str)
+        test:is(parsed_syms, #str, "all characters are parsed")
```

Comment:
I suppose it is better to use single quotes for the consistency:
```sh
$ grep -F "'" -R test/app-tap/datetime.test.lua | wc -l
914
$ grep -F '"' -R test/app-tap/datetime.test.lua | wc -l
301
```

Feel free to ignore.

Reply:
Fixed:

```diff
diff --git a/test/app-tap/datetime.test.lua b/test/app-tap/datetime.test.lua
index bcd772c170..abd09f8295 100755
--- a/test/app-tap/datetime.test.lua
+++ b/test/app-tap/datetime.test.lua
@@ -514,7 +514,7 @@ test:test("Multiple tests for parser (with nanoseconds)", function(test)
         local str, epoch, nsec, tzoffset, check
         str, epoch, nsec, tzoffset, check = unpack(value)
         local dt, parsed_syms = date.parse(str)
-        test:is(parsed_syms, #str, "all characters are parsed")
+        test:is(parsed_syms, #str, 'all characters are parsed')
         test:is(dt.epoch, epoch, ('%s: dt.epoch == %d'):format(str, epoch))
         test:is(dt.nsec, nsec, ('%s: dt.nsec == %d'):format(str, nsec))
         test:is(dt.tzoffset, tzoffset, ('%s: dt.tzoffset == %d'):format(str, tzoffset))
@@ -542,7 +542,7 @@ test:test("Check parsing of full supported years range", function(test)
     for _, y in ipairs(valid_years) do
         local txt = ('%04d-06-22'):format(y)
         local dt, parsed_syms = date.parse(txt)
-        test:is(parsed_syms, #txt, "all characters are parsed")
+        test:is(parsed_syms, #txt, 'all characters are parsed')
         test:isnt(dt, nil, dt)
         local out_txt = tostring(dt)
         local out_dt = date.parse(out_txt)
@@ -2378,7 +2378,7 @@ test:test("Parse strptime format and check with :format()", function(test)
     test:plan(#formats * 2)
     for _, tc in pairs(formats) do
         local dt, parsed_syms = date.parse(tc.buf, {format = tc.fmt})
-        test:is(parsed_syms, #tc.buf, "the whole buffer is parsed")
+        test:is(parsed_syms, #tc.buf, 'the whole buffer is parsed')
         local dt_formatted = dt:format(tc.fmt)
         local buf = tc.buf
         if tc.fmt == '%e' then
```